src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -317,3 +331,43 @@ test('renders form and user can fill it and submit', async () => {
     });
   });
 });
+
+test('render and focuses error notification correctly', async () => {
+  render(<CreateEnrolmentPage />, {
+    mocks: mock3,
```

Comment:
It's hard to know what is mocked inside `mock3`. I think that either rename with better names, or use `describe` -> `it` -tests so that the corresponding `const mocks` is initialized inside describe closure (and describe description perhaps can tell what is mocked). I'm not sure if I was able to tell it in understandable way :)